### PR TITLE
docs: minor doc fixes

### PR DESCRIPTION
**Pull request by marbar3778**
- minor doc fixes that i ran into while reading things
- test if we have github actions

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Updated CHANGELOG_PENDING.md

### CODE REVIEW

File: docs/tendermint-core/metrics.md [hunk 1]
```diff
@@ -34,21 +34,21 @@ The following metrics are available:
 | consensus_rounds                       | Gauge     | 0.21.0    |               | Number of rounds                                                       |
 | consensus_num_txs                      | Gauge     | 0.21.0    |               | Number of transactions                                                 |
 | consensus_total_txs                    | Gauge     | 0.21.0    |               | Total number of transactions committed                                 |
-| consensus_block_parts                  | counter   | on dev    | peer_id       | number of blockparts transmitted by peer                               |
-| consensus_latest_block_height          | gauge     | on dev    |               | /status sync_info number                                               |
-| consensus_fast_syncing                 | gauge     | on dev    |               | either 0 (not fast syncing) or 1 (syncing)                             |
+| consensus_block_parts                  | counter   | 0.25.0    | peer_id       | number of blockparts transmitted by peer                               |
```
Comment (marbar3778) [inline docs/tendermint-core/metrics.md]: put 0.25 here as i couldn't find in the changelog when they were added. I can spend sometime to find the exact releases if people want it but will take a bit of time

File: docs/architecture/adr-015-crypto-encoding.md [hunk 2]
```diff
@@ -21,23 +21,23 @@ representation of the pubkey.
...
 
 The reason for continuing to use amino however is that people can create code
 more easily in languages that already have an up to date amino library.
 It is possible that this will change in the future, if it is deemed that
-requiring amino for interacting with tendermint cryptography is unneccessary.
+requiring amino for interacting with Tendermint cryptography is unnecessary.
 
 The arguments for space efficiency here are refuted on the basis that there are
-far more egregious wastages of space in the SDK.
+far more egregious wastage's of space in the SDK.
```
Comment (erikgrinaker) [inline docs/architecture/adr-015-crypto-encoding.md]: This change is incorrect.

File: docs/tools/README.md [hunk 3]
```diff
@@ -7,8 +7,18 @@ parent:
...
-- [Benchmarking](./benchmarking.md)
-- [Monitoring](./monitoring.md)
+- [Benchmarking](#benchmarking)
 - [Validation of remote signers](./remote-signer-validation.md)
+- [Testnets](#testnets)
+
+
+## Benchmarking
+
+Benchmarking is done tm-load-test, for information on how to use the tool please visit the docs: https://github.com/interchainio/tm-load-test
```
Comment (erikgrinaker) [inline docs/tools/README.md]: "done with"?

File: docs/tools/README.md [hunk 4]
```diff
@@ -7,8 +7,18 @@ parent:
...
+
+
+## Testnets
+
+Tendermint Test Networks is aimed at testing Tendermint with different configurations. For more information please visit: https://github.com/interchainio/testnets.
```
Comment (erikgrinaker) [inline docs/tools/README.md]: "are aimed"?

File: docs/architecture/adr-015-crypto-encoding.md (hunk 2 above)
Comment (marbar3778) [reply docs/architecture/adr-015-crypto-encoding.md]: hm okay will revert, spell check in editor was throwing me the error

Comment (erikgrinaker) [reply docs/architecture/adr-015-crypto-encoding.md]: Yeah, the plural s-form of wastage is a bit esoteric, so not surprised the spell checker isn't pleased, but it's the correct form. Could always rewrite the sentence to be a bit simpler, but 🤷‍♀️.

File: docs/tools/README.md (hunk 4 above)
Comment (marbar3778) [reply docs/tools/README.md]: was basing off the title in the readme but reworded to merely testnets, thanks 👍

File: docs/architecture/adr-015-crypto-encoding.md [hunk 5]
```diff
@@ -35,9 +35,9 @@ increasing the space attached to each validator / account.
 
 The alternative to using amino here would be for us to create an enum type.
 Switching to just an enum type is worthy of investigation post-launch.
-For referrence, part of amino encoding interfaces is basically a 4 byte enum
+For reference, part of amino encoding interfaces is basically a 4 byte enum
 type definition.
-Enum types would just change that 4 bytes to be a varuint, and it would remove
+Enum types would just change that 4 bytes to be a variant, and it would remove
```
Comment (melekes) [inline docs/architecture/adr-015-crypto-encoding.md]: I believe varuint is correct

Comment (marbar3778) [reply docs/architecture/adr-015-crypto-encoding.md]: i went looking for a definition to link to, came to here: https://developers.google.com/protocol-buffers/docs/encoding#signed-integers but not sure if its correct. @ValarDragon do you have something to link here?

Comment (ValarDragon) [reply docs/architecture/adr-015-crypto-encoding.md]: It should be a `varint`, my bad!

This is something thats slightly confusing, but `varints` do support the full unsigned range. The way they handle negatives is to just take the two's complement encoding, and treat it as a uint of the same size.

Comment (marbar3778) [reply docs/architecture/adr-015-crypto-encoding.md]: thank you 😄

**Pull request merged**